src/style.css

```diff
@@ -1,3 +1,103 @@
-body {
+head {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.main-header {
+  width: 100%;
+  max-width: 1200px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+  background-color: #fff;
+  position: fixed;
+  font-size: 15px;
+  margin: 0 auto;
+  box-sizing: border-box;
+  font-weight: 500;
+}
+
+.logo img {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.nav-menu {
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 0;
+  padding-left: 0;
+  height: 60px;
+  width: 650px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #000;
+  margin-left: 20px;
+  position: relative;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  text-align: justify;
+  white-space: nowrap;
+  font-size: 12px;
+  font-weight: 500px;
```

Comment:
The `font-weight` property should be a number, not a number with 'px'. It should be `font-weight: 500;`.